specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/backuprestore.json

```diff
@@ -12,7 +12,7 @@
     "parameters": [
       {
         "name": "vaultBaseUrl",
-        "description": "The vault name, for example https://myvault.vault.azure.net.",
+        "description": "The vault name, for example https://myvault.managedhsm.azure.net.",
```

Comment:
Please also update description of entire command. "This operation is supported only by Managed HSM service only."
https://github.com/Azure/azure-rest-api-specs/blob/002ecc5c02f5f1bc852602b67cc475b2ce42da1b/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/backuprestore.json#L40